editoast/src/views/timetable/paced_train.rs

```diff
@@ -832,6 +839,57 @@ mod tests {
         );
     }
 
+    #[rstest]
+    async fn paced_train_exception_simulation() {
+        let (app, infra_id, train_schedule_id) =
+            app_infra_id_paced_train_id_for_simulation_tests().await;
+        let request = app.get(
+            format!("/paced_train/{train_schedule_id}/simulation/?infra_id={infra_id}&exeption_key=created_exception_key").as_str(),
```

Comment:
This test case doesn't test what you want, because it didn't help catch any problem, in this case, the typo!

```suggestion
            format!("/paced_train/{train_schedule_id}/simulation/?infra_id={infra_id}&exception_key=created_exception_key").as_str(),
```

So the test probably need to be modified to test correctly that you actually processed an exception.

Reply:
I saw you fixed the typo... but what about this?

> So the test probably need to be modified to test correctly that you actually processed an exception.

Because if the test didn't catch the typo, then the test doesn't assert the correct thing.